jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridView.java

```diff
@@ -195,7 +195,7 @@ public void loadListTable(final String key,
                 listTable = newListGrid;
 
                 reloadColumnSortList();
-                
+                listTable.getGridPreferencesStore().setPreferenceKey(null);
```

Comment:
@cristianonicolai yes, that was the idea, only persist when a filter is saved, otherwise.. it's just work in memory. It was to avoid to persist data that never would be used

Reply:
but then I think there are two points here, one that we actually change the current behaviour. As changes are persisted whenever the table preferences change. And the other issue is that once create a filter, you cant really change the column preferences, I mean you can, but it would be persisted right? Users would need to create a new filter to store the new column preferences.

Reply:
ok. so discussed with @jstastny-cz that seems to be the actual behaviour we want.